src/inmanta/logging.py

```diff
@@ -805,8 +811,6 @@ def format(self, record: logging.LogRecord) -> str:
     def _wrap_record(self, record: logging.LogRecord) -> logging.LogRecord:
```

Comment:
We could even lift the logic from `_get_logger_name_for`:
```
        if not self._keep_logger_names:
        ....
```
In here and then call into `_get_logger_name_for` only when required

Reply:
thx!